ajax/dropdown_taskjob.php

```diff
@@ -31,7 +31,7 @@
  * ---------------------------------------------------------------------
  */
 
-if (strpos(filter_input(INPUT_SERVER, "PHP_SELF"), "dropdown_taskjob.php")) {
+if (strpos(filter_input(INPUT_SERVER, "REQUEST_URI"), "dropdown_taskjob.php")) {
```

Comment:
For URLs specific to the plugin, should not we rather use `plugin_glpiinventory_script_endswith` function?

Reply:
Anyways, this is not blocking for current PR

Reply:
`plugin_glpiinventory_script_endswith` is searching in `front` dir, so it would require to change how it works.